### PR TITLE
Update to 5.32.1

### PR DESCRIPTION
**Pull request by cedricziel**
#### Summary

Update the download URL to 5.32.1

### CODE REVIEW

**Review by stylianosrigas — APPROVED** [review]
Thanks @cedricziel 🚀

**Comment by abdullah-alhabshi** [comment]
Hey there team,

If I may ask on behalf of a customer that is currently using this repo for their R&D on Mattermost hosted on `Heroku`, is the PR merge finalized for the update? As seen above it is already approved but have not been updated yet. Appreciate if you guys can provide the latest update on this. Thanks!

File: app.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "MATTERMOST_DOWNLOAD_URI": {
         "description": "The URI to the compiled Mattermost binary you would like to run. Defaults to latest release. See https://www.mattermost.org/download/ for download links.",
         "required": true,
-        "value": "https://releases.mattermost.com/5.26.0/mattermost-5.26.0-linux-amd64.tar.gz"
+        "value": "https://releases.mattermost.com/5.32.1/mattermost-5.32.1-linux-amd64.tar.gz"
```
Comment (cpanato) [inline app.json:13]: we have released 5.35.3, maybe you can update to that

Comment (jasonblais) [reply app.json:13]: @cpanato I wonder if we'd merge this, and the version can be updated with another PR?

Comment (cpanato) [reply app.json:13]: yes we can....

**Review by cpanato — APPROVED** [review]

**Pull request merged**